### PR TITLE
Export Modal

### LINKED ISSUE #115: Export modal
Export should open a modal where users can choose between PNG/SVG.

### PR DESCRIPTION
**Pull request by ydlamba**
**Initial look:**

screenshot_20180807_073153

Fixes #115.

### CODE REVIEW

File: src/components/header/renderer.tsx [hunk 1]
```diff
@@ -418,6 +433,56 @@ class Header extends React.Component<Props, State> {
...
+              this.exportViz('svg');
+              closePortal();
+            }}
+          >
+            <div>
+              <Map />
+              <span>SVG</span>
+            </div>
+            <p>
+              SVG (Scalable Vector Graphics) is a vector image file format which uses geometric forms to represent
```
Comment (domoritz) [inline src/components/header/renderer.tsx]: vector ... vector

tighten text here

File: src/components/header/renderer.tsx [hunk 2]
```diff
@@ -418,6 +433,56 @@ class Header extends React.Component<Props, State> {
...
+              different parts of the image as discrete objects.
+            </p>
+          </button>
+        </div>
+        <div className="user-notes">
+          <h4>Notes</h4>
+          <ol>
+            <li>To get a PDF, export SVG followed by saving as PDF from the print window provided by browser.</li>
```
Comment (domoritz) [inline src/components/header/renderer.tsx]: To get a PDF, you can export and SVG which you save as PDF from the print window of your browser.

File: src/components/header/renderer.tsx [hunk 3]
```diff
@@ -418,6 +433,56 @@ class Header extends React.Component<Props, State> {
...
+            <li>
+              PNGs cannot be enlarged without distortion occurring while SVGs remain crisp and clear at any resolution
```
Comment (domoritz) [inline src/components/header/renderer.tsx]: this should be in the png button

File: src/components/header/renderer.tsx [hunk 4]
```diff
@@ -418,6 +433,56 @@ class Header extends React.Component<Props, State> {
...
+              this.exportViz('png');
+              closePortal();
+            }}
+          >
+            <div>
+              <Image />
+              <span>PNG</span>
+            </div>
+            <p>
+              PNG (Portable Network Graphics) is a raster or bitmap image file format which is made up of a fixed number
```
Comment (domoritz) [inline src/components/header/renderer.tsx]: not sure whether the second part of the sentence is needed

**Comment by ydlamba** [comment]
**After updates:**

screenshot_20180807_082247

**Pull request merged**